languages.yaml

```
@@ -0,0 +1,2572 @@
+Ada:
+  type: programming
+  ascii: |
+    {0}                              *
+    {0}                              *
+    {0}                            * **
+    {0}   *                        *****
+    {0}   **                       ******
+    {0}    **                   ###********
+    {0}    ***          -******#'###*********
+    {0}     *****               #### *********
+    {0}      ****************************{1}XXX{0}**
+    {0}     ** ******************{1}XXXX{2}o{1}X{0}*{1}X{2}o{1}XXX{0}*
+    {0}      **** ***************{1}XXXX{2}ooooo{1}XX{0}*
+    {0}        *******************{1}XXXX{2}ooooo{1}X
+    {0}          *******************{1}XXX{2}ooo{1}X
+    {0}          ************************{1}XX{0}*
+    {0}             *************    ******
+    {0}  /| |                           ****
+    {0} /-|(|(|                     ********
+    {0}                        ************
+    {0}Time-tested, safe     *************
+    {0}and secure            ***********
+    {0}                         *****
+  colors:
+    ansi:
+      - default
+      - cyan
+      - blue
+    rgb:
+      - [255, 255, 255]
+      - [0, 24, 201]
+      - [12, 10, 124]
+    chip: [2, 248, 140]
```

```suggestion
    chip: Rgb[2, 248, 140]
```
We should maybe keep the `Rgb` prefix everywhere to make it more self explanatory and obvious we're talking about colors.

---

Is that valid YAML? AFAIK that would deserialize into a string, which would require some additional parsing in addition to deserialization in `build.rs`.
It could perhaps be an object, like
```yaml
chip:
  r: 2
  g: 248
  b: 140
```
but I was trying to reduce the length of the config while maintaining human readability. Or maybe change the key to `chip-rgb`.

---

Great idea! we're getting closer to the GitHub linguist format: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

`hex`  sounds  good to me.

---

@o2sh done 👍 https://github.com/o2sh/onefetch/pull/699/commits/9f02ad73bb59d84c9254b253e4406d51e95cb8b6